docs/source/model_design/risk.rst

```diff
@@ -13,9 +13,140 @@ Modeling Risk Exposure
 What is a risk factor?
 ----------------------
 
+What is a risk exposure?
+------------------------
```

Comment:
Are `risk.rst` and `risk_factors.rst` supposed to be in this PR, or was that an accident?

Reply:
This was an accident I believe as a result of a merge conflict-like issue from a recent PR that restructured the risk factor documentation. I removed the substantive changes from this PR (the info that previously appeared in the risk.rst in this PR is all in the risk_factors.rst on the main branch)